ts/components/core/Divider/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { View } from "react-native";
+import { IOColors, IOThemeContext } from "../variables/IOColors";
+
+type DividerOrientation = "vertical" | "horizontal";
+
+type DividerProps = {
+  orientation: DividerOrientation;
+};
+
+const DEFAULT_BORDER_SIZE = 1;
+
+/**
+Native `Divider` component
+@param  {SpacerOrientation} orientation 
+ */
+const BaseDivider = ({ orientation }: DividerProps) => (
+  <IOThemeContext.Consumer>
+    {theme => (
+      <View
+        style={{
+          ...{
+            backgroundColor: IOColors[theme["divider-default"]]
+          },
+          ...(orientation === "vertical" && {
+            width: DEFAULT_BORDER_SIZE
+          }),
+          ...(orientation === "horizontal" && {
+            height: DEFAULT_BORDER_SIZE
+          })
+        }}
+      />
+    )}
+  </IOThemeContext.Consumer>
+);
```

Comment:
What do you think about using a function to compose the style to improve code readability?
For example:
```ts
const toDividerStyle = (orientation: DividerOrientation, theme: IOTheme) => {
  const baseStyle = {
    backgroundColor: IOColors[theme["divider-default"]]
  };

  const orientationStyle =
    orientation === "vertical"
      ? { width: DEFAULT_BORDER_SIZE }
      : { height: DEFAULT_BORDER_SIZE };

  return { ...baseStyle, ...orientationStyle };
};

const BaseDivider = ({ orientation }: DividerProps) => (
  <IOThemeContext.Consumer>
    {theme => <View style={toDividerStyle(orientation, theme)} />}
  </IOThemeContext.Consumer>
);
```
What do you think about that?

Reply:
It's definitely better. Already updated :)